TAF/testScenarios/functionalTest/V2-API/app-service/secrets/POST.robot

```diff
@@ -53,7 +53,7 @@ Setup Suite for App Service Secrets
     ...             AND  Setup Suite for App Service  http://${BASE_URL}:48101
 
 Get AppService Token
-    ${command}=  Set Variable  docker exec app-service-${edgex_profile} cat /tmp/edgex/secrets/appservice-${edgex_profile}/secrets-token.json
+    ${command}=  Set Variable  docker exec app-service-${edgex_profile} cat /tmp/edgex/secrets/app-service-${edgex_profile}/secrets-token.json
```

Comment:
IMO, we should not use `edgex_profile` name here since it is the service name, not the profile that we are specifying. 
I recommend using `service_name` instead and set it above to the complete name. i.e:
```
    #EDGEX_SECURITY_SECRET_STORE: "true"
    ...             Set Suite Variable  ${service_name}  app-service-http-export-secrets
    ...             AND  Setup Suite for App Service  http://${BASE_URL}:48102
    #EDGEX_SECURITY_SECRET_STORE: "false"
    ...             ELSE  Run keywords  Set Suite Variable  ${service_name} app-service-http-export
    ...             AND  Setup Suite for App Service  http://${BASE_URL}:48101
```

And then here changes to:
```
${command}=  Set Variable  docker exec ${service_name} cat /tmp/edgex/secrets/${service_name}/secrets-token.json
```

Reply:
cannot agree more, definitely should use `service name`.

Reply:
Updated.